crates/biome_diagnostics_categories/src/categories.rs

```diff
@@ -123,8 +123,8 @@ define_categories! {
     "lint/nursery/noNodejsModules": "https://biomejs.dev/linter/rules/no-nodejs-modules",
     "lint/nursery/noReExportAll": "https://biomejs.dev/linter/rules/no-re-export-all",
     "lint/nursery/noRestrictedImports": "https://biomejs.dev/linter/rules/no-restricted-imports",
-    "lint/nursery/noSemicolonInJsx": "https://biomejs.dev/linter/rules/no-semicolons-in-jsx",
     "lint/nursery/noSkippedTests": "https://biomejs.dev/linter/rules/no-skipped-tests",
+    "lint/nursery/noSuspiciousSemicolonInJsx": "https://biomejs.dev/linter/rules/no-suspicious-semicolons-in-jsx",
```

Comment:
Is it `no-suspicious-semicolons-in-jsx` or `no-suspicious-semicolon-in-jsx`?

Reply:
Thank you for the comment!
I'll fix it. 👍

Reply:
Fixed in https://github.com/biomejs/biome/pull/2220/commits/4158fe8bf961da862ff4ceb3384ea7ae1958dc46

Please review it again when you have time!